src/types.db

```diff
@@ -118,6 +118,11 @@ io_packets              rx:DERIVE:0:U, tx:DERIVE:0:U
 ipc                     value:GAUGE:0:U
 ipt_bytes               value:DERIVE:0:U
 ipt_packets             value:DERIVE:0:U
+ipt_qlen                value:GAUGE:0:U
+ipt_backlog             value:GAUGE:0:U
+ipt_drops               value:DERIVE:0:U
+ipt_requeues            value:DERIVE:0:U
```

Comment:
Please use the existing *if_tx_packets* type and set the *type instance* to "requeued".